### PR TITLE
fix: adjust some ui

### PR DESCRIPTION
**Pull request by swkatmask**
https://mask.atlassian.net/browse/fw-1529
https://mask.atlassian.net/browse/fw-1530
https://mask.atlassian.net/browse/fw-1531
https://mask.atlassian.net/browse/fw-1532
https://mask.atlassian.net/browse/fw-1533
https://mask.atlassian.net/browse/fw-1535
https://mask.atlassian.net/browse/fw-1539
https://mask.atlassian.net/browse/fw-1541
https://mask.atlassian.net/browse/fw-1544
https://mask.atlassian.net/browse/fw-1547

### CODE REVIEW

File: src/app/(normal)/token/[symbol]/layout.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { useComeBack } from '@/hooks/useComeback.js';
 
 export default function TokenPageLayout({ children }: PropsWithChildren) {
     const params = useParams();
-    const { symbol } = params;
+    const symbol = decodeURIComponent(params.symbol as string);
```
Comment (guanbinrui) [inline src/app/(normal)/token/[symbol]/layout.tsx:10]: symbol cloud be undefined

File: src/app/(normal)/token/[symbol]/not-found.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { Link } from '@/esm/Link.js';
 
 export default function NotFoundToken() {
     const params = useParams();
-    const { symbol } = params;
+    const symbol = decodeURIComponent(params.symbol as string);
```
Comment (guanbinrui) [inline src/app/(normal)/token/[symbol]/not-found.tsx:13]: same here

File: src/components/NFTDetail/DownloadImageButton.tsx [hunk 3]
```diff
@@ -16,13 +16,13 @@ export function DownloadImageButton({ url }: { url: string }) {
 
     return (
         <a
```
Comment (guanbinrui) [inline src/components/NFTDetail/DownloadImageButton.tsx:18]: Use clickable button

File: src/components/NFTDetail/DownloadImageButton.tsx [hunk 4]
```diff
@@ -16,13 +16,13 @@ export function DownloadImageButton({ url }: { url: string }) {
...
-            className="flex cursor-pointer select-none items-center gap-1 rounded-full border border-line bg-lightBg px-2 py-1 text-[10px] leading-[14px]"
+            className="flex cursor-pointer select-none items-center gap-1 rounded-full border border-line bg-lightBg px-2 py-1 font-inter text-xs leading-[14px] hover:bg-primaryBottom"
             href={blobUrl}
             target="_blank"
             download={url}
         >
             <DownloadIcon className="h-3 w-3" />
-            <Trans>Download Image</Trans>
+            <Trans>Download image</Trans>
```
Comment (guanbinrui) [inline src/components/NFTDetail/DownloadImageButton.tsx:25]: Image

File: src/app/(normal)/token/[symbol]/layout.tsx (hunk 1 above)
Comment (swkatmask) [reply src/app/(normal)/token/[symbol]/layout.tsx:10]: No, it won't, it's in `token/[symbol]/layout.tsx`

File: src/components/NFTDetail/DownloadImageButton.tsx (hunk 3 above)
Comment (swkatmask) [reply src/components/NFTDetail/DownloadImageButton.tsx:18]: use anchor element to **download**

File: src/components/NFTDetail/DownloadImageButton.tsx (hunk 4 above)
Comment (swkatmask) [reply src/components/NFTDetail/DownloadImageButton.tsx:25]: It's on purpose. see https://mask.atlassian.net/browse/FW-1547

File: src/app/(normal)/token/[symbol]/layout.tsx (hunk 1 above)
Comment (guanbinrui) [reply src/app/(normal)/token/[symbol]/layout.tsx:10]: What if open URL:  `/token/`?

Comment (guanbinrui) [inline src/app/(normal)/token/[symbol]/layout.tsx:10]:
```suggestion
    const symbol = params.symbol ? decodeURIComponent(params.symbol) : '';
```

File: src/app/(normal)/token/[symbol]/not-found.tsx (hunk 2 above)
Comment (guanbinrui) [inline src/app/(normal)/token/[symbol]/not-found.tsx:13]:
```suggestion
    const symbol = params.symbol ? decodeURIComponent(params.symbol) : '';
```

File: src/app/(normal)/token/[symbol]/layout.tsx (hunk 1 above)
Comment (swkatmask) [reply src/app/(normal)/token/[symbol]/layout.tsx:10]: `/token/` won't hit this route. Even it does, `symbol` still would never be `undefined`.

The best way to find out is by giving it a try. [https://firefly-mask-social-git-fix-ui-dimension-dev.vercel.app/token/](https://firefly-mask-social-git-fix-ui-dimension-dev.vercel.app/token/)

Comment (swkatmask) [reply src/app/(normal)/token/[symbol]/layout.tsx:10]: image
image

symbol will never be `undefined` or even empty string

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**